lxc/utils/sort.go

```diff
@@ -36,18 +36,18 @@ func (a StringList) Less(i, j int) bool {
 	return sortorder.NaturalLess(a[i][x], a[j][x])
 }
 
-// ByName represents the type for sorting by one string (i.e. Instance names).
-type ByName [][]string
+// SortColumnsNaturally represents the type for sorting by one string (i.e. Instance names).
```

Comment:
Can you please update the comment to more accurately describe how this type implements `sort.Interface`? Thank you.

Reply:
Would the following comment be appropriate?

```
SortColumnsNaturally represents the type for sorting columns in a natural order from left to right.
```

Reply:
Sounds good.